accounts/ethstore/src/account/crypto.rs

```diff
@@ -159,29 +160,31 @@ impl Crypto {
 #[cfg(test)]
 mod tests {
 	use ethkey::{Generator, Random};
-	use super::{Crypto, Error};
+	use super::{Crypto, Error, NonZeroU32};
+
+	const ITERATIONS: NonZeroU32 = unsafe { NonZeroU32::new_unchecked(10240) };
```

Comment:
Although this is technically safe, by ideology it's probably better to not use any unsafe anywhere.

Reply:
> by ideology it's probably better to not use any unsafe anywhere

Well, I don't think we should treat "don't use unsafe" as a dogma. The are different kinds of unsafe code. Here the unsafety of `new_unchecked` comes from `const_fn` (and probably `const_panic`) being unstable. 
And, in this case, for literals and constants it's easy to reason about the safety.

Nevertheless, I've eliminated all the calls to `new_unchecked`.